src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,11 @@
+package duke;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    @Test
+    public void parseCommands_getCommand_success() {
```

Comment:
Consider following the convention for writing testing methods: featureUnderTest_testScenario_expectedBehavior(). For this case, maybe can consider changing it into parseCommands_validTodoInput_success()